src/Configuration/Provisioning/DashboardDataTable/DashboardTableLinks.jsx

```diff
@@ -0,0 +1,119 @@
+import { Hyperlink, Icon, IconButton } from '@edx/paragon';
+import { getConfig } from '@edx/frontend-platform';
+import { DjangoShort } from '@edx/paragon/icons';
+import PropTypes from 'prop-types';
+import ROUTES from '../../../data/constants/routes';
+
+const { HOME } = ROUTES.CONFIGURATION.SUB_DIRECTORY.PROVISIONING;
+
+export const PlanIdHyperlink = ({ row }) => {
+  const planRowUuid = row.values.uuid;
+  if (getConfig().FEATURE_CONFIGURATION_EDIT_ENTERPRISE_PROVISION) {
+    return (
+      <Hyperlink
+        destination={`${HOME}/${planRowUuid}/view`}
+        key={`edit-${planRowUuid}`}
+        variant="muted"
+      >
+        {planRowUuid}
+      </Hyperlink>
+    );
+  }
+
+  return planRowUuid;
```

Comment:
nit: Given the hyperlink for 3 out of the 4 rows is largely the same except for the text, we could shorten these up with a helper method.
```suggestion
  dashboardLink = (planRowUuid, title) => {
    if (getConfig().FEATURE_CONFIGURATION_EDIT_ENTERPRISE_PROVISION) {
      return (
        <Hyperlink
          destination={`${HOME}/${planRowUuid}/view`}
          key={`edit-${planRowUuid}`}
          variant="muted"
        >
          {title}
        </Hyperlink>
      );
    }
    return title;
  }

  return dashboardLink(row.values.uuid, row.values.uuid);
  }
  
```

Reply:
This was a great suggestion. I updated my PR to incorporate this. Thank you!